apollo-client/src/main/java/com/ctrip/framework/apollo/internals/AbstractConfig.java

```diff
@@ -68,6 +71,7 @@ public AbstractConfig() {
     m_configVersion = new AtomicLong();
     m_arrayCache = Maps.newConcurrentMap();
     allCaches = Lists.newArrayList();
+    propertiesFactory=ApolloInjector.getInstance(PropertiesFactory.class);
```

Comment:
please format this line, i.e `propertiesFactory = ApolloInjector.getInstance(PropertiesFactory.class);`